psw/ae/aesm_service/source/utils/CMakeLists.txt

```diff
@@ -35,6 +35,13 @@ add_library(utils SHARED ${_srcs}
     "../../../../../common/src/sgx_memset_s.cpp"
     "../../../../../common/src/crypto_cmac_128.cpp")
 
+if(NOT "${USE_PREBUILT_OPENSSL}" STREQUAL "0")
```

Comment:
Looks some build scripts are also required the similar changes, such as qpl, qcnl, etc.

Reply:
Confirmed with Xun, seems we can't link prebuilt openssl for qcnl.

Reply:
It looks not the fact. Actually, right now, qpl and qcnl are linking the prebuilt OpenSSL:
https://github.com/intel/linux-sgx/blob/master/psw/ae/aesm_service/source/qpl/CMakeLists.txt#L36
https://github.com/intel/linux-sgx/blob/master/psw/ae/aesm_service/source/qcnl/CMakeLists.txt#L37
https://github.com/intel/linux-sgx/blob/master/psw/ae/aesm_service/source/qcnl/pck_cert_selection/CMakeLists.txt#L50

The motivation for this PR is to switch to system OpenSSL in the untrusted components for some conditions. I think we need to change these scripts to enable the switching on these components.

Reply:
`qcnl` and `qpl` in `aesm_service` can be simply removed. They are built out and packaged in DCAP.

Reply:
Done